### ISSUE TITLE
A11y_Code.visualstudio.com_Docs User Guide_Refactoring_Screenreader:Screenreader announcing two control type for the 'edit' link.

### ISSUE THREAD

**Issue by SwatiTiwari-TCS** [microsoft/vscode-docs#3720]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-manai%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

 

GitHubTags:#A11y_Code.visualstudio.com_Benchmark;#A11ySev2;#A11yTCS;#A11yMAS;#Benchmark;#ChromiumEdge;#Chrome;#DesktopWeb;#MAS1.3.1;#MAS4.1.2;#Win10;#Visual Studio Code;#Narrator;#CAITestPass_May2020;

 

**Environment Details:**
Application Name: code.visualstudio.com
#URL:https://code.visualstudio.com
Windows Version: Windows10
Browser version: Microsoft Edge Version 84.0.516.1 (Official build) dev (64-bit)

**Repro Steps:**
1. Launch the application Url :https://code.visualstudio.com
2. Navigate to "Docs" link.
3. From left navigation, select "User Guide".
4. Navigate to "Refactoring" link.
5. Turn on screen reader and navigate to 'edit' link using CapsLock+RightArrow Key and Observe the issue.

**Actual:**
Start screenreader and navigate using Caps+Arrow key to 'edit link' then Screen reader is announcing two control type for the single the 'edit' control as first its announce edit link and when we again press caps+Arrow key then its announce 'edit button'.

This issue has been Observed in Chromium Egde +Narrator only.

**This issue is observe throughout the application whereever the 'edit link' present.**

**Expected:**
Screenreader should announce correct and only once for the edit control as link because when we activate the edit control then its redirect to another page.

**Attachment link for the Reference:**
https://microsoft-my.sharepoint.com/:v:/p/v-rojad/Eayd8cjwXHZKsKc7O92_L60BcrOSZLjGLoh8NDu30GrBSA?e=yLGhHG

**Comment by JacksonKearl** [comment]
Seems like this mode goes through every DOM element, regardless of whether it's even focusable (the link in this case is not). I notices that this mode will also duplicate readings for all our TOC entries... I don't think there's much we can do about it on our side, this seems like the intended behaviour of the mode.

**Comment by ManishaC1** [comment]
Verified the issue in below environment.
#URL:https://code.visualstudio.com/
Issue is fixed.
Github tags: #Closed
![Screenshot (441)](https://user-images.githubusercontent.com/105198219/217181289-78319d77-0529-4bc4-a769-ed94a786f35f.png)
